### PR TITLE
Fix error on complex example

### PR DESCRIPTION
**Pull request by tteke**
**What**: Fixing an error on readme

**Why**: In the complex example catch already sets error to ```error.message```. So in render, it should just show ```state.error```.

**How**: Render function updated to just show ```state.error``` instead of ```state.error.message```.

**Checklist**:

- [x] Documentation added to the
      [docs site](https://github.com/alexkrolick/testing-library-docs)
- [ ] Tests N/A
- [ ] Typescript definitions updated N/A
- [x] Ready to be merged
      
Yes. Either that or catch shoud update error on the state as ```error: error``` instead of ```error: error.message```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ function Login() {
         </div>
         <button type="submit">Submit{state.loading ? '...' : null}</button>
       </form>
-      {state.error ? <div role="alert">{state.error.message}</div> : null}
+      {state.error ? <div role="alert">{state.error}</div> : null}
```
Comment (dkarski) [inline README.md]: Hi, I have a question about the above syntax.

Is it some convenience to handle show/hidden a component like in README.md?
`{ isTrue ? <Component /> : null }` than `{ isTrue && <Component /> }` (shorter a way)

Comment (kentcdodds) [reply README.md]: I could write a blog post about this for the number of times I talk about it 😅

Basically I've been bitten by this before:

```
{contacts.length && <RenderContactList contacts={contacts} />}
```

If `contacts.length` is `0` then you'll render `0`. No good. Shipped that to prod at PayPal. So I never do that and instead I always go with the more explicit version:

```
{contacts.length ? <RenderContactList contacts={contacts} /> : null}
```

But you do whatever you like :)

**Comment by kentcdodds** [comment]
@all-contributors please add @tteke for docs

**Comment by tteke** [comment]
Thanks

**Pull request merged**